Sources/Elasticsearch/Model/Update.swift

```diff
@@ -1,17 +1,21 @@
 public struct UpdateDoc<T: Encodable> : Encodable {
     public let doc: T
     public let docAsUpsert: Bool
+    public var retryOnConflict: Int? = nil
```

Comment:
This can still be a constant. It becomes a required param in the initiliazer, so update the initializer in the Bulk API

```--- a/Sources/Elasticsearch/Client/ElasticsearchBulk.swift
+++ b/Sources/Elasticsearch/Client/ElasticsearchBulk.swift
@@ -147,7 +147,7 @@ public class ElasticsearchBulk {
         requestBody.append(10)
 
         // Add the document to the request body followed by a newline character (newline -> 10)
-        let updateDoc = UpdateDoc(doc: doc, docAsUpsert: docAsUpsert)
+        let updateDoc = UpdateDoc(doc: doc, docAsUpsert: docAsUpsert, retryOnConflict: retryOnConflict)
```

Reply:
Done